docs/reporting/key-concepts.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Key Concepts
+description: Geocortex Reporting - Learn about key concepts of Geocortex Reporting
+hide_table_of_contents: true
```

Comment:
We have occasionally disabled the TOC when it didn't make sense for the page. In this case i think we can remove this option as the headers make sense IMO.